core/src/js/components/collection/cards.component.ts

```diff
@@ -18,18 +18,18 @@ export const DEFAULT_CARD_HEIGHT = 240;
 	selector: 'cards',
 	styleUrls: [`../../../css/component/collection/cards.component.scss`, `../../../css/global/scrollbar.scss`],
 	template: `
-		<div class="cards">
-			<div class="show-filter">
+		<div class="cards" *ngIf="{ cards: cards$ | async } as value">
+			<div class="show-filter" [ngStyle]="{ 'display': _searchString ? 'none' : 'flex' }">
```

Comment:
I think we need to show the filters, as they influence the search results

Reply:
Yes, of course, this is me rushing.